packages/modules/notification/src/models/notification.ts

```diff
@@ -3,7 +3,7 @@ import { NotificationProvider } from "./notification-provider"
 
 // We probably want to have a TTL for each entry, so we don't bloat the DB (and also for GDPR reasons if TTL < 30 days).
 export const Notification = model.define("notification", {
-  id: model.id({ prefix: "noti" }),
+  id: model.id().primaryKey("noti"),
```

Comment:
suggestion: I would have expected the `prefix` to go on the `id` property, not the `primaryKey`. I think it's a bit strange to couple the concept of a primary key with a prefix, WDYT?

Reply:
Ohhh yeah. I thought the prefix is for the index of primary key. My bad. Let me fix it